specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/examples/Job/FineTuningJob/list.json

```diff
@@ -0,0 +1,89 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "test-rg",
+    "workspaceName": "my-aml-workspace",
+    "api-version": "2024-07-01-preview",
+    "$skipToken": "string",
+    "jobType": "string",
+    "tag": "string",
+    "listViewType": "All",
+    "assetName": "string",
+    "scheduled": false,
+    "scheduleId": "string"
```

Comment:
these are not defined as parameters or query params. Are these meant to be properties?

Reply:
or you are missing this: 
{
            "in": "query",
            "name": "assetName",
            "description": "Asset name the job's named output is registered with",
            "type": "string"
          },
          {
            "in": "query",
            "name": "scheduled",
            "description": "Indicator whether the job is scheduled job.",
            "type": "boolean"
          },
          {
            "in": "query",
            "name": "scheduleId",
            "description": "The scheduled id for listing the job triggered from",
            "type": "string"
          }
in jobs_list in mfe.json